docs/static/settings/monitoring-settings.asciidoc

```diff
@@ -50,6 +50,11 @@ Controls how often data samples are collected and shipped on the Logstash side.
 Defaults to `10s`. If you modify the collection interval, set the 
 `xpack.monitoring.min_interval_seconds` option in `kibana.yml` to the same value.
 
+`xpack.monitoring.collection.write_direct.enabled`::
+
+Forces the internal collectors to write directly to the monitoring {es} without
+traversing the monitoring exporter provided by {es}. Disabled by default, this setting
+is intermediate step in deprecating the internal collectors.
```

Comment:
Adding to my [comment](https://github.com/elastic/logstash/pull/11541#discussion_r376105959) with the code changes, I suspect we are also going to need to do some more extensive documentation changes for this update to mirror the [docs](https://www.elastic.co/guide/en/beats/filebeat/current/monitoring.html) for filebeat monitoring, where there are three options for monitoring: Legacy collection (monitoring data sent via production cluster), Internal collection (monitoring data sent direct to monitoring cluster) and metricbeat collection.

Reply:
I've changed a little bit the documenatiton part related to xpack monitoring, let me know what do you think @karenzone